app-backup/kfoldersync/kfoldersync-3.0.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+KDE_GCC_MINIMAL="4.9"
```

Comment:
Thank you for pointing that out. I think I'll add a gcc version check even in CMakeLists.txt in the next release